frontend/src/features/login/queries.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery, UseQueryResult } from 'react-query'
+
+import { SgidProfilesDto } from '~shared/types/auth'
+
+import { ApiError } from '~typings/core'
+
+import { ApiService } from '~services/ApiService'
+
+export const SGID_PROFILES_ENDPOINT = '/auth/sgid/profiles'
+
+export const useSgidProfiles = (): UseQueryResult<
+  SgidProfilesDto,
+  ApiError
+> => {
+  return useQuery<SgidProfilesDto, ApiError>(['sgidProfiles'], async () => {
```

Comment:
```suggestion

export const sgidProfilesKeys = {
  base: ['sgidProfiles'] as const,
}

...
  return useQuery<SgidProfilesDto, ApiError>(sgidProfilesKeys.base, async () => {
```

for consistency with other queries

Reply:
42010a2e0